src/watch/index.js

```diff
@@ -142,7 +110,7 @@ class Task {
 		this.chokidarOptionsHash = JSON.stringify(chokidarOptions);
 
 		this.filter = createFilter(watchOptions.include, watchOptions.exclude);
-		this.deprecations = watchOptions._deprecations;
+		this.deprecations = [...deprecations, watchOptions._deprecations];
```

Comment:
It seems to me that this should actually read `[...deprecations, ...watchOptions._deprecations]`?
If so, maybe filtering for truthy deprecations is not necessary after all?